terraform/modules/google_discovery_engine_restapi/main.tf

```diff
@@ -24,6 +24,14 @@ locals {
 # datastore.
 #
 # API resource: v1alpha.projects.locations.collections.dataStores.schemas
+
+data "restapi_object" "discovery_engine_datastore" {
+  path = "/dataStores/${var.datastore_id}"
```

Comment:
I reckon it would be a bit nicer to pass the fully qualified datastore name in to the module as a variable from here: https://github.com/alphagov/search-v2-infrastructure/blob/d3b0e297aef83bd53d6ee8e686ab93f10b348dfc/terraform/environment/discovery_engine.tf#L7

Reply:
Have refactored to use this approach